ci/artifacts/xline-pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  name: xline
+  namespace: kube-system
+spec:
+  containers:
+    - name: xline
+      image: ghcr.io/liangyuanpeng/xline:ffa824f
```

Comment:
Should i follow the doc https://github.com/xline-kv/Xline/tree/master/doc/quick-start#build-image to build xline image here?

Maybe there are other CIs that have tested building images, but I didn't find any.

cc @bsbds @Phoenix500526

Reply:
https://github.com/xline-kv/Xline/blob/75e51f4fcbb23b75d3b0f31ea69ed40684694f29/.github/workflows/validation.yml#L42

It looks like the image is built here.

Reply:
Yes, it is. We do the same thing in our benchmark:
https://github.com/xline-kv/Xline/blob/75e51f4fcbb23b75d3b0f31ea69ed40684694f29/.github/workflows/benchmark.yml#L36C1-L41C30

Reply:
Thanks for your quickly confirmation.